docs/index.md

```diff
@@ -0,0 +1,121 @@
+# Welcome to the Stravalib Documentation!
+
+::::{grid} 2
```

Comment:
this makes the landing page prettier / easier for a user to navigate

Reply:
<img width="831" alt="Screen Shot 2022-11-26 at 11 37 22 AM" src="https://user-images.githubusercontent.com/7649194/204104406-481e7363-5860-4c38-b5ed-45e31b1ae5bd.png">